app/services/converters/order_detail_to_journal_rows.rb

```diff
@@ -0,0 +1,23 @@
+module Converters
+  class OrderDetailToJournalRows
```

Comment:
I feel like this could have a more descriptive name. This doesn't really imply it's pretty much one-to-one.